object_store/simple_s3.go

```diff
@@ -0,0 +1,84 @@
+package object_store
```

Comment:
can put it in a more generic package, like "common" or something

Reply:
done